contracts/sponsorship/BiconomySponsorshipPaymaster.sol

```diff
@@ -309,6 +313,14 @@ contract BiconomySponsorshipPaymaster is
             revert InvalidSignatureLength();
         }
 
+        // Note: may not need to cache and forward in the context
+        uint256 unaccountedGasCatched = unaccountedGas;
+
+
+        if(unaccountedGasCatched >= userOp.unpackPostOpGasLimit()) {
+            revert PostOpGasLimitTooLow();
+        } 
+
         bool validSig = verifyingSigner.isValidSignatureNow(
```

Comment:
Can external signer be SA?
-  if yes, suggest using `isValidSignatureNowCalldata` 
- if no, we should probably switch to ecrecover

Reply:
it's always an EOA. can do plain ecrecover

Reply:
done